zebra/zserv.h

```diff
@@ -134,6 +134,9 @@ struct zserv {
 	/* Redistribute default route flag. */
 	vrf_bitmap_t redist_default[AFI_MAX];
 
+	/* VRF registration information. */
+	vrf_bitmap_t vrf_reg_info;
```

Comment:
This only controls the advertisement of interface messages. I'd prefer to call it `ifinfo` like it was originally.

Reply:
It is used for VRF interfaces only, so is the reason for name change as Donald suggested.

Reply:
I respectfully disagree:
* zebra is calling `vrf_bitmap_check(client->ridinfo, vrf_id)` before sending router-id information to the client daemons;
* zebra is calling `vrf_bitmap_check(client->redist[afi][type], vrf_id)` before sending routes to the client daemons;
* zebra is calling `vrf_bitmap_check(client->redist_default[afi], vrf_id)` before sending default-routes to the client daemons;
...and
* zebra (with these changes) is calling `vrf_bitmap_check(client->vrf_reg_info, vrf_id)` before sending interface messages to the client daemons (regardless if the interface belongs to the default VRF or not).

So `zserv->>vrf_reg_info` is actually controlling the distribution of interface information. That's why I think the old `ifinfo` name is more appropriate and less likely to confuse other developers in the future.